test/tests/deformation/mesh-velocity-areas/pipe.i

```diff
@@ -0,0 +1,200 @@
+[GlobalParams]
+  use_displaced_mesh = true
+  displacements = 'disp_x_o disp_y_o disp_z_o'
+[]
+
+[Mesh]
+  type = FileMesh
+  file = pipe.exo
+  parallel_type = replicated
+[]
+
+[Variables]
+  [temp]
+    initial_condition = 0.0
+    order = FIRST
+  []
+[]
+
+[Kernels]
+  [heat]
+    type = HeatConduction
+    variable = temp
+  []
+  [heat_source]
+    type = CoupledForce
+    variable = temp
+    v = 0.0
+  []
+[]
+
+[Materials]
+  [thermal]
```

Comment:
I think you can delete this material, if you change `HeatConduction` to `Diffusion`. Because I don't think anywhere in this input file are you needing a thermal conductivity in order to compute a flux (at least I don't see it anywhere).